packages/xo-web/src/common/plans.js

```diff
@@ -0,0 +1,7 @@
+export const XOA_PLAN_FREE = 1
+export const XOA_PLAN_STARTER = 2
+export const XOA_PLAN_ENTERPRISE = 3
+export const XOA_PLAN_PREMIUM = 4
+export const XOA_PLAN_SOURCES = 5
+
+export default +process.env.XOA_PLAN
```

Comment:
@pdonias Don't we have something similar already?

Sorry if it's the tenth time I ask about this :wink:

Reply:
We have this:
https://github.com/vatesfr/xen-orchestra/blob/7db573885b095255634662fc84a220d4972dcdd9/packages/xo-web/src/common/utils.js#L150-L165